unittest/src/fake/console/tests.rs

```diff
@@ -0,0 +1,44 @@
+use crate::fake;
+use crate::RoAllowBuffer;
+use libtock_platform::share;
+use libtock_platform::DefaultConfig;
+
+// Tests the command implementation.
+#[test]
+fn command() {
+    use fake::SyscallDriver;
+    let console = fake::console::Console::new();
```

Comment:
You should be able to refer to `fake::console::Console` as just `fake::Console` (see the re-export in `unittest/src/fake/mod.rs`), unless I am mistaken.

Reply:
Changed.